src/utils/init/netlify-toml-template.js

```diff
@@ -0,0 +1,16 @@
+const makeNetlifyTOMLtemplate = ({ command, publish }) => `# example netlify.toml
+[build]
+  command = "${command}"
+  functions = "lambda" #  netlify-lambda reads this
```

Comment:
I'm torn between `functions` vs `lambda` as the default here. https://github.com/search?o=desc&p=6&q=functions+filename%3Anetlify.toml&s=indexed&type=Code

Reply:
ok i'll switch it